translator/translate/otel/pipeline/prometheus/translators.go

```diff
@@ -16,10 +17,14 @@ func NewTranslators(conf *confmap.Conf) common.PipelineTranslatorMap {
 	if conf.IsSet(LogsKey) {
 		destinations = append(destinations, common.CloudWatchLogsKey)
 	}
-	if conf.IsSet(MetricsKey) {
+	if conf.IsSet(prometheusremotewrite.AMPSectionKey) {
 		destinations = append(destinations, common.AMPKey)
 	}
 
+	if !conf.IsSet(common.ConfigKey(common.MetricsKey, common.MetricsDestinationsKey)) || conf.IsSet(common.ConfigKey(common.MetricsKey, common.MetricsDestinationsKey, common.CloudWatchKey)) {
+		destinations = append(destinations, common.CloudWatchKey)
+	}
+
```

Comment:
There's a function in `common` that will get the destinations for you. It's currently used in the `jmx` pipeline.

https://github.com/aws/amazon-cloudwatch-agent/blob/de987cd83f7aa68932cf7f1870b8342a94a6cb4d/translator/translate/otel/pipeline/jmx/translators.go#L15